### ISSUE TITLE
Can we get a redirect from WAI pages to the relevant WCAG homepage?

### ISSUE THREAD

**Issue by fstrr** [w3c/wcag#4501]
I've lost count of the amount of times I've hacked off the end of a WCAG 2.x URL thinking I'd get to the relevant version's homepage, but all I get is an "insufficient access priviledges" page.

Current situation:
1. I'm on a page in a 2.x version of WCAG. For example, WCAG 2.2's Name, Role, Value: https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html
2. I hack off the URL, leaving this: https://www.w3.org/WAI/WCAG22/
3. I press Enter to get the browser to go to the page and I get an "insufficient access priviledges" message.

What I'd like to happen:
1. I'm on a page in a 2.x version of WCAG. For example, WCAG 2.2's Name, Role, Value: https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html
2. I hack off the URL, leaving this: https://www.w3.org/WAI/WCAG22/
3. I press Enter to get the browser to go to the page and a redirect sends me to https://www.w3.org/TR/WCAG22/

**Comment by bruce-usab** [comment]
If, for whatever reason, there is a reason not to redirect to TR space from WAI space, https://www.w3.org/WAI/standards-guidelines/#wcag2 would be an acceptable substitute.  I agree that "insufficient access privileges" isn't helpful.
